lib/Gateway.js

```diff
@@ -1157,7 +1183,12 @@ Gateway.prototype.discoverDevice = function (node, hassDevice) {
         hassDevice.object_id = sanitizeId(hassDevice.object_id)
 
         // Set a friendly name for this component
-        payload.name = `${nodeName}_${hassDevice.object_id}`
+        payload.name = getEntityName(
+          node,
+          undefined,
```

Comment:
I think that instead of undefined here you could use the zwaveNode deviceClass? Or something else, othetwise when using devices there could be repeated object_id in entity name, maybe this should be specified in the UI

```
{label: , propertyKey: , propertyName: }
```

Reply:
This part concerns only the customDevices.js(on)

I'm trying to find a case of duplicate. as custom devices are limiting you to specific values. I cannot think of any
in the other hand, you cannot have duplicates between different devices.

Reply:
What about simply don't print values if them are missing? I mean, if valueId is missing we should not fallback to object_id IMO